CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/Heart_Snapshot.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Heart Snapshot</title>
+</head>
+
+<body>
+
+<div class="container">
+    <div class="content">
+        <h1>Heart Snapshot</h1>
+        <h2>What is this app?</h2>
+        <p>
+        Like a doctor’s stethoscope, this app can measure your heart rate. Instead of listening to heart beats, this app can “see” your heart rate. It uses the flash and camera right up against your finger to measure. Together with exercise, this app can measure heart rate recovery. This is similar to a measuring your VO2max. This app was developed as a part of the All of Us Research Program. Your measurements will be used for health research.
+        </p>
+        
+        <h2>What am I being asked to do?</h2>
+        
+        <p>
+        You are being asked to take “snapshots” of your heart health. A snapshot has two parts. One part is measuring your heart at rest. The other part is measuring how your heart recovers after exercise. Everything in this research study is optional. You can stop at any time.
+        </p>
+        
+        <h2>How long does a snapshot take?</h2>
+        
+        <p>
+        Measuring a resting heart rate takes about 1 minute. Measuring heart rate recovery has two options. One option takes 3 minutes and has you go up and down a stair step. The other option takes 12 minutes and has you run or walk.
```

Comment:
I thought we were no longer supporting the 12 minute run test...?

Reply:
TBD. I have a question out about that. This is slightly modified from the Figma docs to make it more appropriate for general-purpose use.